{"name": "test_01", "status": "failed", "statusDetails": {"message": "AssertionError: assert 1 == 2", "trace": "@allure.severity('critical')\n    @allure.feature('模块3：xxxxx')\n    def test_01():\n        '''用例1的描述：xxxx'''\n        print('测试用例1')\n>       assert 1==2\nE       assert 1 == 2\n\ntest_x.py:13: AssertionError"}, "description": "用例1的描述：xxxx", "attachments": [{"name": "stdout", "source": "4026a27d-a211-4135-88df-4a6b054b7b4a-attachment.txt", "type": "text/plain"}], "start": 1595700023940, "stop": 1595700023940, "uuid": "98a1368f-6b7d-42ef-9afd-94dcd45dac93", "historyId": "209a5198088eee57bf0ee8790dfb2514", "testCaseId": "8b287888448907456630b478095e9365", "fullName": "test_x#test_01", "labels": [{"name": "severity", "value": "critical"}, {"name": "feature", "value": "模块3：xxxxx"}, {"name": "suite", "value": "test_x"}, {"name": "host", "value": "kingving"}, {"name": "thread", "value": "18556-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_x"}]}